src/wxflow/configuration.py

```diff
@@ -93,18 +94,20 @@ def print_config(self, files: Union[str, bytes, list]) -> None:
         pprint(config, width=4)
 
     @classmethod
-    def _get_script_env(cls, scripts: List) -> Dict[str, Any]:
+    def _get_script_env(cls, scripts: List, **kwargs) -> Dict[str, Any]:
         default_env = cls._get_shell_env([])
-        and_script_env = cls._get_shell_env(scripts)
+        and_script_env = cls._get_shell_env(scripts, **kwargs)
         vars_just_in_script = set(and_script_env) - set(default_env)
         union_env = dict(default_env)
         union_env.update(and_script_env)
         return dict([(v, union_env[v]) for v in vars_just_in_script])
 
     @staticmethod
-    def _get_shell_env(scripts: List) -> Dict[str, Any]:
+    def _get_shell_env(scripts: List, **kwargs) -> Dict[str, Any]:
         varbls = dict()
-        runme = ''.join([f'source {s} ; ' for s in scripts])
+        # Construct shell variables to parse the config files with
+        runme = ''.join([f'export {key}="{val}" ; ' for key, val in kwargs.items()])
+        runme += ''.join([f'source {s} ; ' for s in scripts])
```

Comment:
Actually, there may be another solution.  I may be able to set the variables and not export them at source time.  i.e.

```suggestion
        prefix = ''.join([f'{key}="{val}" ' for key, val in kwargs.items()]) if len(kwargs > 0) else ''
        runme = ''.join([f'{prefix} source {s} ; ' for s in scripts])
```